renderer_posix_test.go

```diff
@@ -0,0 +1,81 @@
+// +build !windows
+
+package survey
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	expect "github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRenderer_countLines(t *testing.T) {
+	t.Parallel()
+
+	termWidth := 72
+	stdout := new(bytes.Buffer)
+	c, _, err := vt10x.NewVT10XConsole(expect.WithStdout(stdout))
+	vt10x.ResizePty(c.Tty(), termWidth, 30)
```

Comment:
This test is marked as POSIX-only because it allocates a pseudo-terminal to emulate a scenario when the terminal is of a specific width. If anyone knows of a more lightweight approach to generate a terminal at a file descriptor that pretends to be a certain size, I'm all ears!